api_server/src/main/java/io/zetch/app/domain/reply/ReplyDto.java

```diff
@@ -0,0 +1,18 @@
+package io.zetch.app.domain.reply;
+
+import java.io.Serializable;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@Setter
+@SuperBuilder
+@NoArgsConstructor
+public class ReplyDto implements Serializable {
```

Comment:
I think replace the builder class using record class is cleaner in Controller and testing code 

https://github.com/astrohsy/ZETCH/blob/aa56fa3c2f5b78dc1d91b62af905d9a94d8d7d19/api_server/src/main/java/io/zetch/app/domain/user/UserGetDto.java#L7-L13